### ISSUE TITLE
[EgoVenture]: Custom notification system

### ISSUE THREAD

**Issue by ReichNick** [deep-entertainment/issues#29]
Feature is to change Map notification to custom for other purposes.

**Comment by dploeger** [comment]
Implemented in branch issue-29.

The new API can be used like this:

```gdscript
Notification.notify(preload("res://path/to/icon/texture.png"), preload("res://path/to/audio/file.ogg"))
```

The map notification was updated to use this new API, so the change is backwards compatible and doesn't break things.

@ReichNick  Can you please download the [branch of the example game](https://github.com/deep-entertainment/egoventure-example-game/archive/refs/heads/issue-29.zip) and play around with it and test it?

Thanks!

**Comment by dploeger** [comment]
(I will add documentation for this feature once the feature is in the develop branch)

**Comment by dploeger** [comment]
Merged in Develop.

**Comment by dploeger** [comment]
Fixed in 2.2.0
